server/paymentProviders/stripe/creditcard.ts

```diff
@@ -82,15 +82,16 @@ const createChargeAndTransactions = async (
     });
   }
 
-  paymentIntent = await stripe.paymentIntents.confirm(paymentIntent.id, {
-    stripeAccount: hostStripeAccount.username,
-  });
+  paymentIntent = await stripe.paymentIntents.confirm(
+    paymentIntent.id,
+    { payment_method: stripePaymentMethod.id },
+    { stripeAccount: hostStripeAccount.username },
+  );
```

Comment:
While working on this issue, I stumbled upon the fact that trying to re-use a payment intent after a failed verification led to this error (that has [occurrences](https://sentry.io/organizations/open-collective/issues/3058463744) in production):

> GraphQLError: Internal Payment error (invalid PaymentIntent)

The root message from Stripe was:

> You cannot confirm this PaymentIntent because it's missing a payment method. To confirm the PaymentIntent with cus_xxxxxxxxx, specify a payment method attached to this customer along with the customer ID.

Simply passing the `payment_method` here solved the issue and resulted in a successful payment, even after failing the 1st 3Dsecure verification.